data/thecentrons.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Divya Prakash Pandey",
+  "type": "personal",
+  "bio": "IET Lucknow '26 CSE Hare Krishna!",
+  "links": [
+    {
+      "name": "Follow me on Youtube",
+      "url": "https://www.youtube.com/@TheCentrons",
+      "icon": "FaYouTube"
```

You probably be using an invalid icon here. It should be `FaYoutube`. Rest looks good to me! :tada: 

```suggestion
      "icon": "FaYoutube"
```